Update service_overrides.md (#34255)
DataDog/documentation: content/en/tracing/guide/service_overrides.md

### What is an override?
Both integration overrides and service overrides change the service name of spans. The initial service name is referred to as the [**base service**](#base-service).

### Integration overrides
Datadog tracing libraries automatically set different service names on client spans to represent databases, queues, or third-party dependencies in integrations. These types of overrides are referred to as **integration overrides**. With inferred entities, integration overrides are no longer necessary to represent dependencies, and may pollute service lists and maps. For step-by-step instructions on how to remove integration overrides, see [Integration Override Removal][2]

### Service overrides
The service name on spans can be set manually by users. This gives you visibility into specific components of the service, such as shared libraries and middleware layers. These types of overrides are referred to as **service overrides**.

## How overrides are represented
Integration overrides and service overrides are represented similarly in APM.

#### In service and resource pages

> Services that are overrides are flagged as such in the service page header. On hover, find the list of base services where the service name is overridden.

Reviewer: ```suggestion
Services that are overrides are flagged in the service page header. Hover over the flag to see the list of base services where the service name is overridden.
```
